### PR TITLE
add list of saved trips

### PR DESCRIPTION
**Pull request by zoebarth**
image

Create list of saved trips with name and edit and share button.

### CODE REVIEW

File: frontend/src/saved-trips/SavedTrip.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      {name}
+      <div>
+        <a href="/" title="Edit" className="icon">
+          <FontAwesomeIcon icon={faPencilAlt} />
+        </a>
+        <a href="/" title="Share" className="icon">
+          <FontAwesomeIcon icon={faShare}/>
+        </a>
+      </div>
+    </ListGroup.Item>
```
Comment (Cindyzhang977) [inline frontend/src/saved-trips/SavedTrip.js:23]: Should we add an icon for just viewing the trip and not editing?

Comment (zoebarth) [reply frontend/src/saved-trips/SavedTrip.js:23]: Good question! We can make the list item actionable so that if you just click the name you can view it

Comment (zoebarth) [reply frontend/src/saved-trips/SavedTrip.js:23]: Let's discuss at standup tho

Comment (genedwards) [reply frontend/src/saved-trips/SavedTrip.js:23]: +1 for making the name (ideally the whole bar to the left of the pencil icon) clickable

File: frontend/src/saved-trips/SavedTrips.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * Creates list of saved trips.
+ */
+function SavedTrips({trips}) {
+  return (
+    <div className={styles.container}>
+      <h3>Saved Trips:</h3>
+      <ListGroup>
+        {trips.map(trip => (
+          <SavedTrip name={trip}/>
+        ))}
```
Comment (mecolmg) [inline frontend/src/saved-trips/SavedTrips.js:16]: To make this cleaner, you could split it into a helper method, or store this in a variable.

File: frontend/src/saved-trips/SavedTrips.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ListGroup from 'react-bootstrap/ListGroup';
+import styles from './SavedTrips.module.css';
+import SavedTrip from './SavedTrip.js';
+
+/** 
+ * Creates list of saved trips.
+ */
+function SavedTrips({trips}) {
```
Comment (mecolmg) [inline frontend/src/saved-trips/SavedTrips.js:9]: Can you add a comment describing the type of trips? Assuming right now it's an array of strings.

**Pull request merged**